### PR TITLE
absl_flag: convert gflags to absl_flag

### PR DESCRIPTION
**Pull request by lx18233184051**
left gflags will be processed tomorrow, simulation also need to update.

### CODE REVIEW

**Comment by storypku** [comment]
> left gflags will be processed tomorrow, simulation also need to update.

Can you please make clear the reason for the switch from gflags to absl flags ?

**Comment by lx18233184051** [comment]
> > left gflags will be processed tomorrow, simulation also need to update.
> 
> Can you please make clear the reason for the switch from gflags to absl flags ?

absl has some feature，such as support vector of string

ABSL_FLAG(std::vector<std::string>, languages,
          std::vector<std::string>({"english", "french", "german"}),
and support time

ABSL_FLAG(absl::Duration, timeout, absl::Seconds(30), "Default RPC deadline");

File: modules/v2x/v2x_proxy/app/v2x_proxy.cc [hunk 1]
```diff
@@ -57,7 +57,8 @@ V2xProxy::V2xProxy()
   }
 
   hdmap_.reset(new apollo::hdmap::HDMap());
-  std::string map_name = FLAGS_map_dir + "/" + FLAGS_base_map_filename;
+  std::string map_name = absl::GetFlag(FLAGS_map_dir) + "/" +
+                         absl::GetFlag(FLAGS_base_map_filename);
```
Comment (xiaoxq) [inline modules/v2x/v2x_proxy/app/v2x_proxy.cc:61]: Use `absl::StrCat` for all such cases.
https://abseil.io/tips/3

Comment (lx18233184051) [reply modules/v2x/v2x_proxy/app/v2x_proxy.cc:61]: ok

File: modules/v2x/v2x_proxy/app/v2x_proxy.cc [hunk 2]
```diff
@@ -57,7 +57,8 @@ V2xProxy::V2xProxy()
   }
 
   hdmap_.reset(new apollo::hdmap::HDMap());
-  std::string map_name = FLAGS_map_dir + "/" + FLAGS_base_map_filename;
+  std::string map_name = absl::StrCat(absl::GetFlag(FLAGS_map_dir), "/",
+                                      absl::GetFlag(FLAGS_base_map_filename));
```
Comment (xiaoxq) [inline modules/v2x/v2x_proxy/app/v2x_proxy.cc:61]: Seems like we have a lot of operations on file path.
Maybe we can add a "modules/common/util/file.h" later to provide things similar to python's
```
os.path.join()
os.path.basename()
...
```

**Pull request merged**